android/src/main/java/com/pspdfkit/react/PSPDFKitModule.java

```diff
@@ -42,10 +42,13 @@
     @Nullable
     private Runnable onPdfActivityOpenedTask;
 
-    public PSPDFKitModule(ReactApplicationContext reactContext, Application application) {
+    public PSPDFKitModule(ReactApplicationContext reactContext) {
         super(reactContext);
-        // We register an activity lifecycle callback so we can get notified of the current activity.
-        application.registerActivityLifecycleCallbacks(this);
+        Activity activity = getCurrentActivity();
```

Comment:
This sadly doesn't work since `getCurrentActivity` returns null when starting up the app. So our `setPageIndex` method won't work anymore.